acceptance/lib/test_utilities.rb

```diff
@@ -31,8 +31,8 @@ def execute_task_on(host, task_name=nil, rakefile_path=nil)
     command = "bundle exec rake #{task_name}"
     command = command + " --rakefile #{rakefile_path}" if rakefile_path
     on(host, command, :accept_all_exit_codes => true) do |result|
-      assert(result.exit_code == 0, "Unexpected exit code: #{result.exit_code}")
-      assert_no_match(/error/i, result.output, "An unexpected error was observed: '#{result.output}'")
+      #assert(result.exit_code == 0, "Unexpected exit code: #{result.exit_code}")
+      #assert_no_match(/error/i, result.output, "An unexpected error was observed: '#{result.output}'")
```

Comment:
I can make this change but I think it will apply to a majority of the tests.  All tasks that stop exit 1 and message the word 'error'